### PR TITLE
Update INSTALL.md

### PR DESCRIPTION
**Pull request by spitters**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -181,7 +181,7 @@ The HoTT library requires a custom version of Coq with three modifications: the
 command-line option `-indices-matter`, universe polymorphism, and private types.
```
Comment (SkySkimmer) [inline INSTALL.md]: HoTT doesn't require a custom Coq any more. All these features are in 8.6.

**Comment by spitters** [comment]
Thanks. I made a more drastic change now. @JasonGross @SkySkimmer please let me know if you agree.

File: INSTALL.md [hunk 2]
```diff
@@ -177,59 +177,13 @@ archive requires a working version of the `autoreconf` utility.
...
+The HoTT library requires the command-line option `-indices-matter`. This
+changes the interpretation of equality to one that conforms to the
+homotopy-theoretic interpretation.
 
 ### Configuration of the HoTT library
 
-Next you should configure the HoTT library:
+To configure the HoTT library:
 
 1. If you installed the custom Coq as your default version of Coq,
```
Comment (SkySkimmer) [inline INSTALL.md]: This still talks about "the custom Coq".

Comment (SkySkimmer) [inline INSTALL.md]: There's also one in detailed instructions > prerequisites (line 140).

**Comment by spitters** [comment]
Thanks. I hope I got all of them now.

**Comment by SkySkimmer** [comment]
There's still a couple, please grep for them.
Also, "the Coq" is incorrect grammar.

File: INSTALL.md [hunk 3]
```diff
@@ -177,71 +155,23 @@ archive requires a working version of the `autoreconf` utility.
...
+homotopy-theoretic interpretation.
 
 ### Configuration of the HoTT library
 
-Next you should configure the HoTT library:
+To configure the HoTT library:
 
-1. If you installed the custom Coq as your default version of Coq,
-   which means that it it can be fond in PATH, run
+1. If Coq can be fond in PATH, run
```
Comment (mikeshulman) [inline INSTALL.md]: foUnd

**Comment by spitters** [comment]
@mikeshulman thanks for spotting that. Fixed.

**Pull request merged**